cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -112,16 +101,8 @@ def form_data_single_datastore(request, second_provider, provider):
     return form_data
 
 
-@pytest.fixture(scope="module")
-def enable_disable_migration_ui(appliance):
```

Comment:
this fixture already exists in `v2v.py` as `migration_ui`, which is attached to `v2v_providers` fixture and unnecessary here.